chapters.md

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
You should rebase your branch to squash this into the commits you update

Reply:
> I think a lot of these commits should be a single PR. It will also be easier/faster to review them.

> Do not merge, into your branch. You should rebase from master and edit your patches. It makes it easier to revert patches if necessary.

> The title of this patch Start new Chapters is not very specific nor does it explain what it adds and why.

Yes you are right, but unfortunately I'm to stupid for GitHub to use it in a correct way,
I got a lot of error messages when I use a desktop tool like GitKraken. I had asked some people to help me but no luck.

So I don't know what I have to do. I want delete my Matroska fork and make a new one, but Mosu told me that is not a good idea, because of losing all conversations.

If it is OK, then I will open an issue ticket for each problem.

Reply:
If you're using Windows I suggest using TortoiseGit.  I only use that daily, unless I really have to do something very technical. It make rebasing, reordering and squashing (merging with the previous one) very easy. I will still tell you to resolve conflicts but at least it's in a more intuitive way. I don't think this is possible on github and I wouldn't know how to do it on the command line.

Reply:
I have try it, but it looks more complicate to me. I'm so sorry.

I use now GitHubDesktop. It was easy to create a new repo and now I can test some features and functions. But I don't found "rebasing or squashing".

My new repo is about [Matroksa-Playback](https://github.com/hubblec4/Matroska-Playback) where I try to summarize all info about the possibilities to use chapters or Matroska-Linking for alternate playback. It is in an early stage but a lot of test files are already uploaded.